.github/workflows/bikes.yml

```diff
@@ -47,5 +47,10 @@ jobs:
         
     - name: Helm Install PR
       run: |
-        helm upgrade --install --namespace $GITHUB_HEAD_REF bikesharing-$GITHUB_HEAD_REF samples/BikeSharingApp/Bikes/charts/bikes --set "image.repository=${{ secrets.CONTAINER_REGISTRY }}/bikes,image.tag=$GITHUB_SHA,imagePullSecrets[0].name=${{ secrets.IMAGE_PULL_SECRET }},ingress.hosts[0]=$GITHUB_HEAD_REF.s.${{ secrets.MASTER_SPACE }}.bikesharing.${{ secrets.HOST_SUFFIX }},ingress.annotations.\"kubernetes\\.io/ingress\\.class\"=traefik-azds"
+        helm upgrade --install --namespace $GITHUB_HEAD_REF bikesharing-$GITHUB_HEAD_REF samples/BikeSharingApp/Bikes/charts/bikes \
+          --set image.repository=${{ secrets.CONTAINER_REGISTRY }}/bikes \
+          --set image.tag=$GITHUB_SHA \
+          --set imagePullSecrets[0].name=${{ secrets.IMAGE_PULL_SECRET }} \
+          --set ingress.hosts[0]=$GITHUB_HEAD_REF.s.${{ secrets.MASTER_SPACE }}.bikesharing.${{ secrets.HOST_SUFFIX }} \
+          --set ingress.annotations.kubernetes\\.io/ingress\\.class=traefik-azds
```

Comment:
>kubernetes\\.io/ingress\\ [](start = 36, length = 25)

Just to be sure: we don't need the quotes around that anymore because we're not included in a higher scope string, right?

Reply:
It has to do with the leveling of escaping. I've tested and it works as expected

Reply:
Alright, thanks for confirming

---
In reply to: [333753945](https://github.com/Azure/dev-spaces/pull/168#discussion_r333753945) [](ancestors = 333753945)